doc/release/platforms/README.sgi.rst

```diff
@@ -49,8 +55,7 @@ This can be done using the following settings:
 
```

Comment:
I think it's reasonable to have that attitude, but at the same time I don't think it would be very impressive from an outsider's perspective.  I'm more in favor of saying outright that some of it is wrong or hasn't been updated for years due to a lack of users on that platform or something than having the "may be wrong" statement when we know for sure it is, and am willing to yield to that.  We might also consider moving these platforms into a separate "old platforms" section or something, just so that it's more obvious that they're unlikely to work.